tasks/puppet_restore.sh

```diff
@@ -11,3 +11,4 @@ BOLT=${PUPPET_BIN_DIR}/bolt
 FACTER=/usr/local/bin/facter
 
 $BOLT command run ""$PUPPET_BIN_DIR"/puppet-backup restore "$PT_targetdir"/pe_migrate-$($FACTER -p pe_server_version)_backup.tgz --force" --targets "$PT_targethost" --tty --user root --no-host-key-check --private-key "$PT_privatekey"
```

Comment:
What's the logic of this script?  This is a Bolt task that runs `/opt/puppetlabs/bin/bolt` on the target node to run `puppet-backup restore` on a different node?